public/app/features/profile/UserProfileEditForm.tsx

```diff
@@ -72,16 +72,14 @@ export const UserProfileEditForm = ({ user, isSavingUser, updateProfile }: Props
               />
             </Field>
 
-            <div className="gf-form-button-row">
```

Comment:
Can you also remove this in the `SharedPreferences` component, so it looks the same on the profile page?

Reply:
Actually, while removing it, could you move the Button to be after the `FieldSet` component instead of inside of it? Then it would look exactly the same, just without the SASS class.

Reply:
how are you previewing `SharedPreferences `Component in UI? Can you direct?

Reply:
Hello @tskarhed, moving the Button outside `FieldSet` increases the spacing between Fields and Button. Here is the attached ss of the same. lmk.
![image](https://github.com/grafana/grafana/assets/57278642/19559757-66a0-4ac3-a207-c134af2e6da3)